controls/overlay.rb

```diff
@@ -26,109 +23,76 @@
 
-    Both the holder of a digital certificate and the issuing authority must take careful 
-    measures to protect the corresponding private key. Private keys should always be generated 
-    and protected in FIPS 140-2 validated cryptographic modules.
+    Both the holder of a digital certificate and the issuing authority must take careful measures to protect the corresponding private key. Private keys should always be generated and protected in FIPS 140-2 validated cryptographic modules.
 
-    All access to the private key(s) of MongoDB must be restricted to authorized and 
-    authenticated users. If unauthorized users have access to one or more of MongoDB\'s private 
-    keys, an attacker could gain access to the key(s) and use them to impersonate the database 
-    on the network or otherwise perform unauthorized actions.'
+    All access to the private key(s) of MongoDB must be restricted to authorized and authenticated users. If unauthorized users have access to one or more of MongoDB\'s private keys, an attacker could gain access to the key(s) and use them to impersonate the database on the network or otherwise perform unauthorized actions.'
   end
 
   control 'V-81873' do
-    desc 'The CMS standard for authentication is CMS-approved PKI certificates. Once a PKI 
-    certificate has been validated, it must be mapped to a DBMS user account for the 
-    authenticated identity to be meaningful to MongoDB and useful for authorization decisions.'
+    desc 'The CMS standard for authentication is CMS-approved PKI certificates. Once a PKI certificate has been validated, it must be mapped to a DBMS user account for the authenticated identity to be meaningful to MongoDB and useful for authorization decisions.'
   end
 
   control 'V-81879' do
-    tag "cci": ['CCI-001184']
-    tag "nist": ['SC-23', 'Rev_4']
+    impact 0.0
+    desc "caveat", "Not applicable for this CMS ARS 3.1 overlay, since the related security control is not included in CMS ARS 3.1"
   end
 
   control 'V-81881' do
-    impact 'none'
-    desc 'caveat', 'Not applicable for this CMS ARS 3.1 overlay, since the related security 
-         control is not applied to this system categorization in CMS ARS 3.1'
+    impact 0.0
+    desc "caveat", "Not applicable for this CMS ARS 3.1 overlay, since the related security control is not included in CMS ARS 3.1 for this system categorization"
   end
 
   control 'V-81897' do
-    impact 'none'
-    desc 'caveat', 'Not applicable for this CMS ARS 3.1 overlay, since the related security                                             
-         control is not applied to this system categorization in CMS ARS 3.1'
+    impact 0.0
+    desc "caveat", "Not applicable for this CMS ARS 3.1 overlay, since the related security control is not mandatory in CMS ARS 3.1"
   end
 
   control 'V-81899' do

```

Comment:
@ejaronne This control has one applicable NIST tag so I'm not sure if we need to include it in the overlay